src/main/webapp/index.html

```diff
@@ -2,10 +2,42 @@
 <html>
     <head>
         <meta charset="UTF-8">
-        <title>My Portfolio</title>
+        <title>Create Event</title>
         <link rel="stylesheet" href="style.css">
         <script src="script.js"></script>
     </head>
     <body>
+       
+         <form action="/get-info"method="POST">
+        <h1 class=center>Create Your Event :)!</h1>
+      <div class=row>
+          <div class=column>
```

Comment:
Can you double check your indentation for the rest of this file? There are a few lines that look off to me